pkg/kubelet/cm/container_manager_linux.go

```diff
@@ -377,6 +382,14 @@ func ensureSystemContainer(rootContainer *fs.Manager, manager *fs.Manager) error
 			continue
 		}
 
+		// Get PIDs in kubelet group so we can remove them from the list of
+		// PIDs to move.
+		kubeletPids, err := kubeletContainer.GetPids()
```

Comment:
What about docker container? Won't that also be included if we were to do a recursive walk from `/`?